### ISSUE TITLE
Disable Label Start Position from Create GRG by Area

### ISSUE THREAD

**Issue by Dbarnes1** [Esri/solutions-grg-widget#11]
It looks to me like when you create a GRG based on an Area, you can set whatever base angle you want. This renders the label start position as inaccurate unless you set your base angle at about 90 or 270 degrees. 

I think that, as a result, the Label Start Position option should be disabled from the grid settings page when you Create a GRG by Area. 

To replicate the issue I'm seeing:
1. Navigate to the Create GRG by Area page, and then navigate to the Grid Settings Page
2. In the Grid Settings Page, make sure that the Label Start Position is set to Lower-Left
3. Create a GRG by Area, from your start point drag the base angle at about 90 degrees from north. You'll see the labels appear in the proper location, like so: 
![image](https://user-images.githubusercontent.com/13024357/29735304-e95bbba8-89ac-11e7-8e38-09ccb0240d3d.png)

4. Now, create a GRG by Area and from your start point drag the base angle south (180 degrees). You'll see the label appears in the upper-left instead of its proper location: 
![image](https://user-images.githubusercontent.com/13024357/29735376-5ab8c46c-89ad-11e7-963a-425d076f306a.png)
Or if you do it slightly differently (just over 180), it'll draw in the lower-right. 
![image](https://user-images.githubusercontent.com/13024357/29735418-a3ef2d7e-89ad-11e7-8026-d15fb4684980.png)

Basically, I'm not exactly sure how this works, but the further away you get from drawing your base angle at either 90 or 270 degrees (which appears on the map as relatively flat), this doesn't seem to work.
